src/v2/custom_conversions/organization_create_metric_descriptor.apib

```diff
@@ -0,0 +1,64 @@
+### Step 1. Create a custom metric descriptor [POST /organizations/{org_id}/metric_descriptors]
+
+**Note that this endpoint is identical to the V1 endpoint.
+Its documentation has been copied here for convenience in explaing the custom conversion upload flow.
+AdStage may choose to release a V2 namespaced endpoint with a different feature set and shape in the future.**
+
+Sets up a metric descriptor to allow data upload. Returns the url to allow upload.
```

Comment:
Capitalize URL for consistency throughout the docs.

Reply:
- [x] Replaced all in project.